### ISSUE TITLE
self.structuredClone is not a function in self-hosted edge functions

### ISSUE THREAD

**Issue by sagacity-felix** [supabase/edge-runtime#104]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Our code that contains [`self.structuredClone`](https://deno.land/api@v1.34.0?s=structuredClone) only runs when deployed to Deno Deploy and not when run in the self-hosted edge-runtime.

```
TypeError: self.structuredClone is not a function
    at Server.<anonymous> (file:///home/deno/functions/test-structured-clone/index.ts:57:24)
    at async Server.#respond (https://deno.land/std@0.187.0/http/server.ts:220:24)
````

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create a test function that calls `self.structuredClone`
2. Start the local development Supabase setup (`supabase start`) and serve the functions (`supabase functions serve`)
3. Call the edge function you created

## Expected behavior

This function should be available the same way it is available in Deno Deploy when I deploy the same function using `supabase functions deploy`.

## System information

- OS: macOS
- Browser: chrome and cURL
- Version of Supabase CLI: 1.63.0 (edge runtime version v1.3.5)

**Comment by laktek** [comment]
Reopening the issue until Supabase CLI is updated with the fix.

**Comment by laktek** [comment]
This should be fixed in CLI [v1.64.3](https://github.com/supabase/cli/releases/tag/v1.64.3)
